src/main/java/ca/intelliware/ihtsdo/mlds/config/ReplacementJHipsterReloaderAutoConfiguration.java

```diff
@@ -1,70 +1,69 @@
-package ca.intelliware.ihtsdo.mlds.config;
```

Comment:
@jaykamal if this file isn't needed, please delete.

Reply:
@dmcgihtsdo We have deleted this file. Thanks.